src/transforms/util/pullRequestLinkGenerator.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Generates a create pull request link for GH in the format [URL]/compare/[name]?title=[...keys]-[name]
+ * This format allows the use of title query param to set the PR name
+ */
+export const generateCreatePullRequestUrl = (baseUrl: string, name: string, issueKeys: null | string[] = []) => {
```

Comment:
Is it possible to use a feature-flag in this method, so we can do a gradual rollout (ie, reduce risk)?

Reply:
Can do, created flag use-new-github-pull-request-url-format, will include.